src/routes/utils/jwt.js

```diff
@@ -0,0 +1,43 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import errorResponse from './response-format';
+
+dotenv.config();
+const {
+  SECRET_KEY,
+} = process.env;
+
+const Jwt = {
+  async generateToken(payload) {
+    const token = await jwt.sign(payload, SECRET_KEY, {
+      expiresIn: '14d',
+    });
+    return token;
+  },
+
+  async verifyToken(token) {
+    const decoded = await jwt.verify(token, SECRET_KEY);
+    return decoded;
+  },
+
+  async authToken(req, res, next) {
+    const header = req.headers.authorization;
+    if (typeof header === 'undefined') {
+      return errorResponse(res, 401, 'You are not authorized to make this action');
+    }
+
+    const token = header.split(' ')[1];
+    if (!token) {
+      return errorResponse(res, 401, 'You are not authorized to make this action please login');
+    }
+    try {
+      const decoded = await jwt.verify(token, process.env.SECRET_KEY);
```

Comment:
This line seems to be doing the same thing as the `verifyToken` method just above.